src/components/AccountDetails/Transaction.tsx

```diff
@@ -1,9 +1,8 @@
-import React from 'react'
```

Comment:
It's a tsx file, this is required, no?

BTW, I think this whole file should be reverted, these changes are necessary for the mod files AFAICT